torch/csrc/jit/script/parse_string_literal.h

```diff
@@ -67,11 +67,14 @@ inline std::string parseStringLiteral(
       case 't':
         c = '\t';
         break;
-      case 'h':
+      case 'x':
         throw ErrorReport(range) << "unsupported hex specifier";
+      case 'u':
+      case 'U':
+        throw ErrorReport(range) << "unsupported unicode specifier";
       default:
-        // \0NN
-        if (auto v = parseOctal(str, pos + 1)) {
+        // octal value in format \nnn, n is [0-7]
+        if (auto v = parseOctal(ret_str, pos)) {
```

Comment:
1) we keep updating ret_str, so we should use ret_str

2) in parseOctal we already handle pos + 1 for '\', so no need to do it here.